packages/grafana-ui/src/components/SimpleSelect/SimpleSelect.tsx

```diff
@@ -0,0 +1,59 @@
+import { useCombobox } from 'downshift';
+import React, { useState } from 'react';
+
+import { SelectableValue } from '@grafana/data';
```

Comment:
SelectableValue is very wide, and has a bunch of extra properties that we're not going to use, at least to begin with.

I think we should make a new interface here, at least to begin with. Maybe something like

```ts
type Value= string | number;
type Option = {
  label: string;
  value: Value;
}
```

and then we go from there

Reply:
True, looking at it, it seems like it was introduced specifically for Select. I think the `[key: string]: any;` kinda defeats its purpose a bit. Let's use this opportunity to lock things down 😄

Reply:
+1 to @joshhunt comment. If we have a chance to rethink the interface, let's do it. However, keep in mind that `SelectableValue` holds certain properties for a reason (i.e. `description` or `imageUrl`) that may be useful if we are aiming at replacing some usages of the Select component.